documentation/modules/auxiliary/scanner/http/log4shell_scanner.md

```diff
@@ -63,6 +63,11 @@ starting with `#` will be treated as comments. Lines may also contain the string
 injection point. This enables query parameters to be included in the request which are required for certain
 applications.
 
+### LEAK_PARAMS
+^-separated list of additional params to leak, for example the following would leak the USER and PATH environment
```

Comment:
Just confirming if the `^-` prefix is needed here 👀 

```suggestion

Separated list of additional params to leak, for example the following would leak the USER and PATH environment
```

Reply:
I think the example helps to clarify a bit, but a more explicit wording could be something like:

A list of additional params to leak, with each separated by the `^` character. For example the following would leak the USER and PATH environment

Reply:
Thanks! :+1: